src/Slider.test.tsx

```diff
@@ -127,24 +129,32 @@ describe('UpsellSlider', () => {
             const clickSpy = jest.fn();
 
             render(<Slider>
-                <span data-testid="1" onClick={clickSpy}/>
+                <span key={ 1 }  data-testid="1" onClick={clickSpy}/>
             </Slider>);
 
             const scrollElement = screen.getByRole('list');
 
-            // eslint-disable-next-line testing-library/prefer-user-event
-            fireEvent.mouseDown(scrollElement);
-            // eslint-disable-next-line testing-library/prefer-user-event
-            fireEvent.mouseMove(scrollElement, { clientX: 100, clientY: 0 });
-            // eslint-disable-next-line testing-library/prefer-user-event
-            fireEvent.mouseUp(scrollElement);
+            act(() => {
+                // eslint-disable-next-line testing-library/prefer-user-event
+                fireEvent.mouseDown(scrollElement);
+                // eslint-disable-next-line testing-library/prefer-user-event
+                fireEvent.mouseMove(scrollElement, { clientX: 100, clientY: 0 });
+                // eslint-disable-next-line testing-library/prefer-user-event
+                fireEvent.mouseUp(scrollElement);
+            });
 
             // This click is normally triggered when releasing the mouse after scrolling
-            await userEvent.click(scrollElement);
+            act(() => {
+                userEvent.click(scrollElement);
```

Comment:
I'm pretty sure userEvent has act built in

Reply:
Also I thought that these userEvent commands were now async?

Reply:
I thought so as well, but when I don't wrap them I get flooded in warnings:

` console.error
      Warning: An update to ForwardRef inside a test was not wrapped in act(...).`

Reply:
Reverted them, since the warnings where already there and are non-blocking.